app/controllers/concerns/sms.rb

```diff
@@ -2,36 +2,16 @@
 require "twilio-ruby"
 
 # A set of utility methods used in relation to SMS notifications
-# @author Eric Schlange <eric.schlange@northwestern.edu>
 module Sms
-  extend ActiveSupport::Concern
-
-  # Send an SMS message body to a specified receiving participant (leaving
-  # off recipient defaults the value to the current participant)
-  # rubocop:disable Metrics/AbcSize
-  def send_sms(recipient = current_user, message_body)
+  def send_sms(recipient, message_body)
+    return unless Rails.env.staging? || Rails.env.production?
```

Comment:
@ericcf what do you mean by WRT?  Thanks for the discussion and links.  In this case, I am not sure how else to handle `return unless Rails.env.staging? || Rails.env.production?` for twilio in the testing and development environment.